integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -706,10 +706,11 @@ fn test_bad_orphan() {
             let chunk = block.mut_chunks()[0].get_mut();
 
             match &mut chunk.inner {
-                ShardChunkHeaderInner::V1(inner) => inner.prev_outcome_root = CryptoHash([1; 32]),
-                ShardChunkHeaderInner::V2(inner) => inner.prev_outcome_root = CryptoHash([1; 32]),
-                ShardChunkHeaderInner::V3(inner) => inner.prev_outcome_root = CryptoHash([1; 32]),
-                ShardChunkHeaderInner::V4(inner) => inner.prev_outcome_root = CryptoHash([1; 32]),
+                ShardChunkHeaderInner::V1(inner) => inner.prev_block_hash = CryptoHash([1; 32]),
```

Comment:
Why did we need to change this from prev_outcome_root to prev_block_hash for the other versions?

Reply:
V5 has no `prev_outcome_root` and I think with `prev_block_hash` the test's meaning doesn't change